website/templates/project/registration_editor_templates.mako

```diff
@@ -154,4 +158,28 @@
     </div>
 </script>
 
+<script>
+ function setAllBoxes(value) {
```

Comment:
Putting the function here pollutes the global scope and is much less maintainable -- so yes, that would be better.